BBHSE/new.php

```diff
@@ -0,0 +1,263 @@
+</select>
+<br><br>
+
+<b>Risk Factors:</b><br>
+<input type=checkbox name='risk_compliance'>&nbsp;<b>Hx of Tx Non-Compliance</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_elope'>&nbsp;<b>Hx/Px of Elopement</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_multi'>&nbsp;<b>Hx of Multiple Dx</b><br>
+<input type=checkbox name='risk_inpatient'>&nbsp;<b>Prior Inpatient Treatment</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_hxsuicide'>&nbsp;<b>Prior Homicide or Suicide Attempt</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_injury'>&nbsp;<b>Self Injurious</b><br>
+<input type=checkbox name='risk_suicide'>&nbsp;<b>Current Suicide Ideation</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_self'>&nbsp;<b>Imminent Risk of Harm to Self</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_threat'>&nbsp;<b>Threats to Harm Others</b><br>
+<input type=checkbox name='risk_aggression'>&nbsp;<b>Aggression</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_homicide_ideation'>&nbsp;<b>Current Homicidal Ideation</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<input type=checkbox name='risk_harm'>&nbsp;<b>Imminent Risk of Harm to Others</b><br>
+<input type=checkbox name='risk_other'>&nbsp;<b>Other Risk Factors Noted</b>
+<br><br>
+
+<b><u>Risk Management</u></b><br><?/* prompt plus default text  */?>
+<textarea cols=120 rows=7 wrap=virtual name="risk_management">The current risk is rated as low based on the factors above and it is my clinical judgment that the recommended level of counseling indicated below is sufficient to manage this risk.</textarea>
+<br><br>
+
+<b><u>Presenting Problems/Symptoms</u></b><i> (Behavioral Stability:  include onset, duration, frequency, and severity.</i><br><?/*  prompt, no default */?>
+<textarea cols=120 rows=7 wrap=virtual name="presenting_problem" ></textarea>
+<br><br>
+
+<b><u>Diagnostic Impressions</u>  </b><i>Include criteria for all diagnosis provided based on clinical interview with primary caretaker/observation of caregiver-child relationship</i><br>
+<textarea cols=120 rows=7 wrap=virtual name="diagnostic_impressions" ></textarea>
+<br><br>
+
+
+
+<b>Findings:</b>
+<br><br>
+<b>Axis I</b><br>
+<textarea cols=120 rows=3 wrap=virtual name="axis1" ></textarea><br><br>
+<b>Axis II</b><br>
+<textarea cols=120 rows=1 wrap=virtual name="axis2" ></textarea><br><br>
+<b>Axis III</b><br>
+<textarea cols=120 rows=1 wrap=virtual name="axis3" ></textarea><br><br>
+<b>Axis IV</b><br>
+<textarea cols=120 rows=1 wrap=virtual name="axis4" ></textarea><br><br>
+<b>Axis V/CGAS</b><br>
+<textarea cols=120 rows=1 wrap=virtual name="axis5" ></textarea><br><br>
+
+<b>Eligibility Criteria</b><br><br>
+<input type=checkbox name='criteria1'>&nbsp;<b>Client has an ICD-9-CM diagnosis in the following range: 294.80, 294.90, 300.00 through 305.90, 307.10, 307.23, 307.50 through 307.70, 308.00 through 312.40, 312.81 through 314.90;</b>
```

Comment:
No...Nor is the Axis system used anymore.  I will update this form immediately.  Never even noticed this!  The clinicians should have said something in the intervening years!  They know what the checkbox means at least.